setup.cfg

```diff
@@ -62,9 +62,8 @@ markers =
     network: mark tests that require internet access
 
 [flake8]
-max-line-length = 88
 exclude = build/*, dist/*, pip_tools.egg-info/*, piptools/_compat/*, .tox/*, .venv/*, .git/*, .eggs/*
-extend-ignore = E203  # E203 conflicts with PEP8; see https://github.com/psf/black#slices
+extend-ignore = E501,E203  # E203 conflicts with PEP8; see https://github.com/psf/black#slices
```

Comment:
you can set the max-line-length to whatever line you created, no need to set it to infinite

Reply:
Good, point. I changed it to 100 rather than dropping it.